operations/utility_storage/CSM_rbd_tool_Usage.md

```diff
@@ -59,3 +59,31 @@ optional arguments:
    /dev/rbd0
    RBD device mounted at - ncn-m002.nmn:/etc/cray/upgrade/csm
    ```
+
+## Troubleshooting the RBD Tool
+
+### Issue
+
+Moving the `rbd` device can fail with the error message:
+
+ `mount: /etc/cray/upgrade/csm: mount(2) system call failed: Structure needs cleaning.`
+
+### Fix
+
+To clean the device, first get the `rbd` device name and location.
+This can be found in the information outputted by the status check shown above. The device name is likely `/dev/rbd0` or similar and it should be located on `ncn-m001` or `ncn-m002`.
+Run the following command on the node that the device is located on.
+
+(`ncn-m#`) Clean the device.
```

Comment:
We might mention that you'll be prompted several times -- and maybe have example output:

```
Group descriptor 7999 checksum is 0x6f65, should be 0x6971.  FIXED.
Pass 1: Checking inodes, blocks, and sizes

Pass 2: Checking directory structure
Pass 3: Checking directory connectivity
Pass 4: Checking reference counts
Pass 5: Checking group summary information
Block bitmap differences:  +(32768--33917) +(98304--99453) +(163840--164989) +(229376--230525) +(294912--296061) +(819200--820349) +(884736--885885) +(1605632--1606781) +(2654208--2655357) +(4096000--4097149) +(7962624--7963773) +(11239424--11240573) +(20480000--20481149) +(23887872--23889021) +(71663616--71664765) +(78675968--78677117) +(102400000--102401149) +(214990848--214991997)
Fix<y>? yes
Free blocks count wrong (246632145, counted=257747999).
Fix<y>? yes
Free inodes count wrong (65529942, counted=65535989).
Fix<y>? yes
Padding at end of inode bitmap is not set. Fix<y>? yes

/dev/rbd0: ***** FILE SYSTEM WAS MODIFIED *****
/dev/rbd0: 11/65536000 files (0.0% non-contiguous), 4396001/262144000 blocks
```